### PR TITLE
Try to get travis pass again

### PR DESCRIPTION
**Pull request by scarabeusiv**

### CODE REVIEW

File: rpmlint/version.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 try:
-    from importlib.metadata import version as implib_metadata_version
+    from importlib.metadata import version
 except ImportError:
-    from importlib_metadata import version as implib_metadata_version
+    from importlib_metadata import version
 
-__version__ = implib_metadata_version('rpmlint')
+__version__ = version(__name__)
```
Comment (Conan-Kudo) [inline rpmlint/version.py]: Is it safe to use the `version()` name like this? And where does `__name__` come from?

Comment (scarabeusiv) [reply rpmlint/version.py]: Nope, i was just trying if it would resolve it.

**Comment by scarabeusiv** [comment]
@Conan-Kudo bigger problem the importlib seem not to work without the egg-info available.

So it works if rpmlint is installed but i would have to catch another exception and set fake version if not...

**Comment by scarabeusiv** [comment]
The error we see is basically the same like https://github.com/pyinstaller/pyinstaller/issues/4258

@mimi1vx do you happen to have any suggestions?

**Comment by Conan-Kudo** [comment]
@scarabeusiv We probably should just have a fake version fallback of `0.0.0`.

**Comment by scarabeusiv** [comment]
Ok added the fallback there.

**Comment by scarabeusiv** [comment]
The importlib will be added on Leap 15 with pytest MU that is being now processed by QA.

**Pull request merged**